bundles/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/model/StationProgram.java

```diff
@@ -1,5 +1,22 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
 package org.openhab.binding.opensprinkler.internal.model;
 
+/**
+ * The {@link StationProgram} class corresponds to the program set in the station.
+ *
+ * @author Chris Graham - Initial contribution
```

Comment:
I think this file was added by @FlorianSW in #5890.


And thanks, Kai. My fault that this was committed. Sorry.

Reply:
Sorry for picking the wrong author - this is fixed now.